### DISCUSSION TITLE
Questions about 3 cascading dynamic dropdown

### DISCUSSION THREAD

**Discussion by plagonell** [sqlpage/SQLPage#665] [Q&A]
Hello, I have problems with 3 dynamic dropdowns, the last one doesn't solve it well, take the multistep form as an example and it didn't work for me
![Captura de pantalla de 2024-11-01 14-55-38](https://github.com/user-attachments/assets/ad39cda6-a1e0-4c74-a2dc-5b3493583baa)
![Captura de pantalla de 2024-11-01 14-55-02](https://github.com/user-attachments/assets/e32b758d-5ca6-4d11-9bbb-1b0b2f97c567)
Excuse me if I'm doing something wrong. 

I use sqlpage for query and graphics and it is fine, very vergatronico.
Thanks

**Comment by lovasoa** [comment]
Hello! I'll be glad to help you solve your issue. Can you please: 
 - copy and paste your sql code here in a way that makes it easy to reproduce your issue locally for others
- explain clearly what behavior you expect
- explain clearly what behavior you observe instead

**Reply by plagonell** [reply]
Hi, after exhaustively reviewing the code, I detected that it was setting the select id variable to '', I removed it and it is working as it should. My apologies, Sqlpage is excellent and each version incorporates incredible features.
